app/Console/Commands/RebuildQueryserviceData.php

```diff
@@ -16,18 +17,20 @@ class RebuildQueryserviceData extends Command
     private const NAMESPACE_PROPERTY = 122;
     private const NAMESPACE_LEXEME = 146;
 
-    protected $signature = 'wbs-qs:rebuild {--domain=*} {--chunkSize=50} {--sparqlUrlFormat=http://queryservice.default.svc.cluster.local:9999/bigdata/namespace/%s/sparql}';
+    protected $signature = 'wbs-qs:rebuild {--domain=*} {--chunkSize=50} {--queueName=manual-intervention} {--sparqlUrlFormat=http://queryservice.default.svc.cluster.local:9999/bigdata/namespace/%s/sparql}';
```

Comment:
does this default value mean we have to make sure to have a api-queue with that name running in case no queue is specified?

Reply:
Yes, this is exactly how it works. Would you think `default` is a better default here?

Reply:
ok thanks for clarifying. hm I'm not sure, depends on how we will want to use the default queue in the future. If we will still use it for day-to-day stuff that gets automatically queued then I'm in favor of this approach because it makes it so visible if jobs were kicked off manually :+1: